src/test/java/com/harragan/battleshipsboot/service/GameServiceTest.kt

```diff
@@ -0,0 +1,47 @@
+package com.harragan.battleshipsboot.service
+
+import com.harragan.battleshipsboot.model.game.Game
+import com.harragan.battleshipsboot.model.game.Player
+import com.harragan.battleshipsboot.repositorys.GameRepository
+import com.nhaarman.mockito_kotlin.mock
+import org.assertj.core.api.Assertions.assertThat
+import org.junit.Before
+import org.junit.Test
+
+import org.mockito.Mockito.`when`
+import org.mockito.MockitoAnnotations
+import java.util.*
+
+
+class GameServiceTestKt {
+
+    private val gameRepository: GameRepository = mock<GameRepository>()
+
+    private val game1 = Game(2,10)
+    private val gameService = GameService(gameRepository)
+    private val player1 = Player("John")
+    private val player2 = Player("Smith")
+    private val player3 = Player("Jones")
+
+    init {
+        MockitoAnnotations.initMocks(this)
```

Comment:
mockitojunitrunner and `@injectmocks`, try and avoid using the kotlin mock library if possible